plugins/rbac-common/src/permissions.ts

```diff
@@ -67,3 +69,34 @@ export const policyEntityPermissions = [
   policyEntityDeletePermission,
   policyEntityUpdatePermission,
 ];
+
+/**
+ * This permission is used to authorize actions that involve reading
+ * catalog entities.
+ */
+export const catalogEntityReadPermission = createPermission({
```

Comment:
you do not need to create this

you can import from the catalog-common plugin

we are doing this in App.tsx
```
import { catalogEntityReadPermission } from '@backstage/plugin-catalog-common/alpha';
```